### DISCUSSION TITLE
How to get path gain not effected by antenna.

### DISCUSSION THREAD

**Discussion by fanyimeng0** [NVlabs/sionna#451] [Q&A] [answered]
Hello, I would like to know how to get the correct path loss without being affected by the antenna. I learnt a lot from [this question ,](https://github.com/NVlabs/sionna/discussions/380#discussioncomment-8993210) but I have found that rotation seems to affect this result. I have found that although when I use iso antennas, using a different rotation causes the result to change. But I don't think this should happen as the rotation of the antenna should not change the path loss values. So I'm curious if there is any way to help me get the true road loss value, thanks. The exact code and results are shown below.

```py
# Configure antenna array for all transmitters
scene.frequency = 2.14e9 # in Hz; implicitly updates RadioMaterials

scene.synthetic_array = True # If set to False, ray tracing will be done per antenna element (slower for large arrays)
from sionna.constants import PI 
scene.remove("rx")
scene.remove("tx")
orientation = tf.constant([0,-PI/2,0])
scene.tx_array = PlanarArray(num_rows=1,
                             num_cols=1,
                             vertical_spacing=0.5,
                             horizontal_spacing=0.5,
                             pattern="iso",
                             polarization="V")

scene.rx_array = PlanarArray(num_rows=1,
                             num_cols=1,
                             vertical_spacing=0.5,
                             horizontal_spacing=0.5,
                             pattern="iso",
                             polarization="V")

tx = Transmitter(name="tx",
                 position=[10,10,30],
                 orientation=orientation
                )

scene.add(tx)

rx = Receiver(name="rx",
              position=[20,20,50],
              )
scene.add(rx)
paths = scene.compute_paths(max_depth=5,
                            num_samples=1e6)  # Number of rays shot into directions defined
                                              # by a Fibonacci sphere , too few rays can
                                              # lead to missing paths
print("Path types: ", paths.types.numpy())
a, tau = paths.cir()
paths.normalize_delays=False
t = tau[0,0,0,:]/1e-9 # Scale to ns
a_abs = np.abs(a)[0,0,0,0,0,:,0]
a_max = np.max(a_abs)
t = np.concatenate([(0.,), t, (np.max(t)*1.1,)])
a_abs = np.concatenate([(np.nan,), a_abs, (np.nan,)])
print(a_abs)
```

Before orientation:
![image](https://github.com/NVlabs/sionna/assets/81957249/027b6798-5cc6-46b2-af64-63878f55bd08)
After orientation:
![image](https://github.com/NVlabs/sionna/assets/81957249/1874b89f-c103-48b9-a70a-ba9066abe882)

**Comment by faycalaa** [comment, marked as answer]
Hi,
The orientation of a single linearly polarized antenna does impact the energy it captures.
Explanations are available in the [primer](https://nvlabs.github.io/sionna/em_primer.html#coordinate-system-rotations-and-vector-fields) and in the [API documentation on polarization models](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.antenna.polarization_model_1).
However, the total energy of the field impinging the receive antenna is not impacted. You can observed this by using a dual-polarized antenna at the receiver, which captures the total energy of the received field and not only a single polarization direction, and by summing the energy of the two polarisation directions.

**Reply by fanyimeng0** [reply]
Thanks for your response.
